__tests__/app.test.js

```diff
@@ -233,4 +233,121 @@ describe('GET /api/reviews/:review_id', () =>
             });
         });
     });
+});
+
+describe('POST /api/reviews/:review_id/comments', () =>
+{
+    test('server responds with status code 201', () =>
```

Comment:
This is a pointless test imo. If the test didn't return a 201 status code it wouldn't matter if separated out this behaviour.

Reply:
Ok, I'll update it.